tests/core/interfaces-gpio-chardev/mk-sim-chips.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+CONFIG_BASE=/sys/kernel/config/gpio-sim
+CHIP_NAME=snaptest
+
+/usr/sbin/modprobe gpio-sim || true
```

Comment:
What's the point of `|| true` if the script does not have `set -e`? I'd drop that.